src/firebase/firebaseDB.ts

```diff
@@ -2,20 +2,30 @@ import {firebase} from "./config";
 
 class DataBase {
     database: firebase.database.Database;
+    serviceTable: any;
```

Comment:
can you make this serviceTable from 'any' to {serviceId: string|null; serviceDetails: any}[] any because .val()in the above statement returns (string, number, or boolean), an array, or an object, or null if u want to specify it to specific u can but any is fine

Reply:
I had it this way idk why I changed it